### ISSUE TITLE
Translation is not visable

### ISSUE THREAD

**Issue by gemguardian** [andrewhancox/moodle-filter_translations#130]
We think it is a chaching issue because we could not replicate on a different device. 

**translation language;** Dutch
**course page**: https://moodle.academy/mod/book/view.php?id=393

**Issue**: translation is done by using "missing on this page". 
When saving the translation is not visable and still shows the English language. 
When I switch to "start in-line translation" I can see the full translation.

What is the case: part of the translation has be done on a different account with start in-line translation and later finished by student. 
But we got another report similar to this issue after the update of last week.

**Comment by andrewhancox** [comment]
Are you testing on the latest version? There was a major bug in some caching work that has since been fixed.

**Comment by rjnl** [comment]
Yes, we have the latest version of filter_translation plugin on the Academy site.
I'll check whats going on @gemguardian .

**Comment by andrewhancox** [comment]
Probably fixed as part of #131
